src/app/App.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+import React from 'react';
+import Greeting from '../components/Greeting';
+
+function App() {
+  return (
+
+    <>
+      <h3>Random Greetings</h3>
+      {/* <Greeting /> */}
+
+    </>
+
+  );
+}
+
+export default App;
```

Comment:
- [ ] You have duplicated the App component. Please remove this file to keep your code clean.